### ISSUE TITLE
Training Setting

### ISSUE THREAD

**Issue by z-jiaming** [TRI-ML/VOST#5]
Thanks for your work!

I want to ask about the training setting. Most previous works training on YouTube-VOS and DAVIS in the main-training after the image pre-training.

But your paper said "All the models are initialized from their best DAVIS’17 checkpoint (usually pre-trained on large-scale image and/or video collection) and ﬁne-tuned on the training set of VOST, unless stated otherwise." and there is NO DAVIS dataset in Fig.5 about training datasets.

Do you remove the DAVIS dataset because you tested that it is better or just because you want to use the DAVIS validation set ( initialized from their best DAVIS’17 checkpoint)?

Thanks a lot for your answer!

**Comment by raghavgoyal14** [comment]
Also, interested in the question. Going by the camera-ready version of the paper, 

- The best reported results on VOST using AOT in Table 2 is $J_{tr}$ 36.4 (presumably using Img + DAVIS)
- But the pre-trained model provided in the repo is "pre-trained on static imges and YouTubeVOS" (link: https://github.com/TRI-ML/VOST/tree/main/aot_plus)
- And lastly, in the Figure 5 (attached), DAVIS dataset isn't mentioned at all.
  - The interesting part is that the model pre-trained on Img (static) only is achieving better performance (close to $J_{tr}$ 38), compared to $J_{tr}$ 36.4

![image](https://github.com/TRI-ML/VOST/assets/2176778/ad8bc4d9-ec9e-4e75-8913-9e59a1241782)

cc @davidhalladay
